src/stravalib/model.py

```diff
@@ -820,6 +906,13 @@ class Lap(
     strava_model.Lap,
     BoundClientEntity,
 ):
+    """An object that represents an Activity lap.
+
+    This object inherits the BoundClient to support API access through lazy
+    methods.
```

Comment:
Seeing this kind of docstring being repeated often, I wonder if we should mention this inheritance of `BoundClient` at all, as it is not really of concern to users of this library. See, e.g., the documentation of [KNeighborsClassifier from scikit-learn](https://scikit-learn.org/stable/modules/generated/sklearn.neighbors.KNeighborsClassifier.html). Its docstring doesn't mention that it inherits from `KNeighborsMixin`, `ClassifierMixin`, or `NeighborsBase`.

Reply:
good point. i think i added it from a contributor perspective (ie it took me some time to understand why we had the pattern of inheritance) BUT i think you are correct - these are more for users so i'll remove mention of BoundClient. sklearn is a great model for us to follow!